bin/vdm

```diff
@@ -67,8 +71,43 @@ try:
     sys.path.insert(0, os.path.join(base_dir, 'lib/python', 'vdm'))
     from voltcli import runner
     from server import HTTPListener
+
 # Be selective about exceptions to avoid masking load-time library exceptions.
 except (IOError, OSError, ImportError), e:
     sys.stderr.write('Exception (%s): %s\n' % (e.__class__.__name__, str(e)))
     sys.exit(1)
-HTTPListener.main(runner, HTTPListener, 8000)
+
+def main():
+    parser = OptionParser(usage="usage: %prog [options] filepath",
+                          version="%prog 1.0")
+    parser.add_option("-p", "--path",
+                  action="store", type="string", dest="filepath")
+    (options, args) = parser.parse_args()
+
+    return options.filepath
+
+
+if __name__ == '__main__':
+  path = main()
+
+app_root = os.path.dirname(os.path.abspath(__file__))
+os.chdir(os.path.normpath(app_root + os.sep + os.pardir) + '/lib/python/vdm')
+os.getcwd()
+
+if path is None:
```

Comment:
We can set the location of vdm.xml file using --path command line option. Other than that, there is no option to select an alternate VDM config home.